src/components/CatalogLayerContextMenu.vue

```diff
@@ -32,15 +41,30 @@
         </li>
       </ul>
     </li>
-    <li v-if="canZoom(layerMenu.layer)" @click.prevent.stop="zoomToLayer(layerMenu.layer)">
+
+    <!-- TODO add item description -->
+    <li v-if="layerMenu.layer.geolayer && layerMenu.layer.visible" class="menu-icon">
+      <layer-opacity-picker
+        @add-layer-menu-item="addLayerMenuItem"
+        @show-layer-menu="showSubMenuContext"
+        :layerMenu="layerMenu"
```

Comment:
replace the `layermenu` property with the singles (and only those) properties needed by the component `layer-opacity-picker` to work (eg: `layer`, `opacity`, `show`, ... )